### DISCUSSION TITLE
Unable to Access Remote Jupyter Server in VS Code

### DISCUSSION THREAD

**Discussion by karthick-govindaraj** [microsoft/vscode-jupyter#14114] [Questions and Answers]
I'm facing an issue while trying to access a remote Jupyter Notebook server using the VS Code Jupyter extension. I have a remote Jupyter server running on a BeagleBone board using the following command:

`jupyter notebook --NotebookApp.allow_origin='*' --NotebookApp.ip='0.0.0.0' --allow-root --no-browser`

The server starts successfully. However, when I try to connect to the server using the URL http://192.168.4.14:8891/tree?token=a0f1c1902c9c3eb2b7e0e002fe10b7488a9f973c2298f0c0 through the VS Code Jupyter extension, I encounter an error. 

![image](https://github.com/microsoft/vscode-jupyter/assets/94033319/99634da7-1591-47ab-816e-98f205e6d21b)

ERROR in terminal
```
[I 2023-08-14 15:36:17.669 JupyterNotebookApp] 302 GET /tree/hub/api (@192.168.4.117) 7.92ms
[I 2023-08-14 15:36:17.682 JupyterNotebookApp] 302 GET /tree/tree? (@192.168.4.117) 1.87ms
[W 2023-08-14 15:36:17.749 JupyterNotebookApp] 404 GET /tree/api/sessions?1692007577689 (90557546c8e14d5eaac759136f8aded7@192.168.4.117) 16.22ms referer=None
[W 2023-08-14 15:36:17.759 JupyterNotebookApp] 404 GET /tree/api/kernelspecs?1692007577657 (65cc61e21948416a95363f8c5a695869@192.168.4.117) 8.80ms referer=None
[W 2023-08-14 15:36:17.765 JupyterNotebookApp] 404 GET /tree/api/kernels?1692007577669 (f08d01e8c4b04f89b1da8ecd622185bc@192.168.4.117) 13.66ms referer=None
[W 2023-08-14 15:36:17.771 JupyterNotebookApp] 404 GET /tree/api/kernels?1692007577679 (65b742a44e5b4a448150acdfc3bce835@192.168.4.117) 18.47ms referer=None
[W 2023-08-14 15:36:17.802 JupyterNotebookApp] 404 GET /tree/api/kernelspecs?1692007577736 (2b65a6c373594a898a53439ec8ad4a6e@192.168.4.117) 12.37ms referer=None
```

but,  I can access it through my browser by using same URL.

suggest me the solution to access jupyter in vs code.

**Comment by wangxk15** [comment]
Same issue here, any solutions so far?

**Comment by DonJayamanne** [comment]
Please could you enable logging as follows:
* Open settings & go into `Jupyter->Logging`
* Change the value to `verbose`
* Reload VS Code,
* Attempt to repro the issue & then copy the output from the `Jupyter` output panel.
